2_dataCleaning/DataCleaning_Code.Rmd

```diff
@@ -162,6 +162,7 @@ table(mydata$Sex)
 When we look at the table for the Sex variable, we see another data entry problem. We have a third category that should really be another case of "Female". We can fix all of these at once, but sometimes messing with factors is more complicated than numeric variables.
 
 ```{r}
+# MVT: is there a way to do this with tidyverse ?
```

Comment:
Try the following:

`mydata <- mydata %>% mutate(Sex = dplyr::recode(Sex, "Femle" = Female))` (with appropriate newline/tab if desired)

Should we introduce the pipe in the programming tutorial? Could be great but also could be too much conceptually. If not, write this without the pipe.